contrib/util/docker/dockers/dev-orthanc-proxy/templates/default.conf.template

```diff
@@ -0,0 +1,18 @@
+server {
+   listen  8080  default_server;
+   
+   location  /orthanc/  {
+      proxy_pass http://${ORTHANC_HOST}:${ORTHANC_PORT};
+      proxy_set_header HOST $host;
+      proxy_set_header X-Real-IP $remote_addr;
+      proxy_set_header Authorization "Basic ${ORTHANC_CREDS}";
```

Comment:
I added credentials in the proxy because it has CORS enabled and it didn't look ideal to set those from a CORS request.